homework/java/zhdanok_liudmila/lecture2/task1/src/test/java/by/jrr/mytests/servise/DogServiseTest.java

```diff
@@ -0,0 +1,48 @@
+package by.jrr.mytests.servise;
+
+import by.jrr.mytests.bean.Dog;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class DogServiseTest {
+
+    @Test
+    public void bark() {
+        Dog dog = new Dog();
```

Comment:
you can use private method to construct Dog

```
private Dog makeDogWithPrice(Double price) {
      Dog dog = new Dog();
      dog.setName("Malysh");
      dog.setPrice(price);
      return dog;
}
```

here you can use makeDogWithPrice(null);